lib/ably/models/paginated_result.rb

```diff
@@ -41,48 +44,57 @@ def initialize(http_response, base_url, client, options = {}, &each_block)
       @items = items.map { |item| yield item } if block_given?
     end
 
-    # Retrieve the first page of results.
-    # When used as part of the {Ably::Realtime} library, it will return a {Ably::Util::SafeDeferrable} object,
-    #   and allows an optional success callback block to be provided.
+    # Returns a new PaginatedResult for the first page of results.
+    #
+    # @spec TG5
+    #
+    # @return [PaginatedResult, Ably::Util::SafeDeferrable]
     #
-    # @return [PaginatedResult,Ably::Util::SafeDeferrable]
     def first(&success_callback)
       async_wrap_if_realtime(success_callback) do
         return nil unless supports_pagination?
         PaginatedResult.new(client.get(pagination_url('first')), base_url, client, pagination_options, &each_block)
       end
     end
 
-    # Retrieve the next page of results.
-    # When used as part of the {Ably::Realtime} library, it will return a {Ably::Util::SafeDeferrable} object,
```

Comment:
I think we shouldn't remove the bit about `SafeDeferrable`

Reply:
Reverted.